app/models/conference_preference.rb

```diff
@@ -0,0 +1,6 @@
+class ConferencePreference < ActiveRecord::Base
+  include GithubHandle
```

Comment:
This module only works if a `user` method is available in the class. With switching the `Attendance`/ `ConferencePreference` to be owned by the team, it will no longer work.